src/components/Table/Table.story.jsx

```diff
@@ -3781,7 +3781,7 @@ storiesOf('Watson IoT/Table', module)
     }
   )
   .add(
-    'Simple Stateful Example with column overflow menu',
+    'Simple Stateful Example with column overflow menu and aggregate column values',
```

Comment:
This is nitpicky, but could you remove the first three words from this to make it easier to read? Theres a ton of table stories and it can be difficult to find certain ones
```suggestion
    'with column overflow menu and aggregate column values',
```